addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js

```diff
@@ -320,14 +335,18 @@ export class Powerbox {
     // -------------------------------------------------------------------------
 
     _filter(term, commands) {
-        const initalCommands = commands;
+        const initalCommands = this.openedInInlineRoot ? commands.filter(c => c.allowInInlineRoot) : commands;
+        if (term === '') {
```

Comment:
Looks like you moved the logic here ?

Reply:
see https://github.com/odoo/odoo/pull/106348#discussion_r1031262397